types/parcel-bundler/index.d.ts

```diff
@@ -173,6 +179,12 @@ declare class ParcelBundler {
     addPackager(type: string, packager: string): void;
 
     bundle(): Promise<ParcelBundler.ParcelBundle>;
+
```

Comment:
Maybe you can add a description

e.g. something like this (or better)

```ts
/**
 * Starts a simple http (or https) server.
 * 
 * @param `https` `{ cert, key }` pointing to the location of key and cert file or `true` to generate a key
 */
```

i copied and adapted from [official web site](https://en.parceljs.org/api.html)